_unittest/test_21_Circuit.py

```diff
@@ -772,6 +792,7 @@ def test_43_create_and_change_prop_text(self):
         assert self.aedtapp.modeler.create_text("text test", "1000mil", "-2000mil")
 
     @pytest.mark.skipif(config["NonGraphical"], reason="Change property doesn't work in non-graphical mode.")
+    @pytest.mark.skipif(is_linux and config["desktopVersion"] == "2024.1", reason="Schematic has to closed.")
```

Comment:
Would i make sense to create a new aedtapp to run this test and `test_45_create_circuit_from_multizone_layout` ?

Reply:
They are different problems.

Test 44 fails because it tryis to modify the text title inside the schematic circuit, and in Linux in non graphical mode is crashing. I will submit a bug.

In test 45, the problem is the EDB object as described here: https://github.com/ansys/pyedb/issues/414